test/unit/payment_icon_test.rb

```diff
@@ -15,7 +15,7 @@ class PaymentIconTest < ActiveSupport::TestCase
   end
 
   test '#bank_transfers returns all icons of group bank_transfers' do
-    assert_equal ['boleto', 'dankort', 'dwolla', 'forbrugsforeningen', 'giropay', 'laser', 'sofort', 'maestro', 'unionpay', 'visaelectron'], PaymentIcon.bank_transfers.pluck(:name)
```

Comment:
Build is broken in master because of https://github.com/activemerchant/payment_icons/commit/215ce3665fe945f4b4bad0a9ac63ceae150ef14a. This is fixing the build.